### PR TITLE
The Start Now verb sets the round to start immediately

### PR DESCRIPTION
**Pull request by coiax**
Previously, if you pressed Start Now while the server was still
initialising, it would complain, and you'd have to wait until the lobby
was ready.

Now pressing it will have the server start as soon as able.

Uses: debugging qol.

### CODE REVIEW

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -513,15 +513,17 @@ var/global/BSACooldown = 0
...
+	if(ticker.current_state == GAME_STATE_PREGAME || ticker.current_state == GAME_STATE_STARTUP)
+		ticker.start_immediately = TRUE
 		log_admin("[usr.key] has started the game.")
-		message_admins("<font color='blue'>[usr.key] has started the game.</font>")
+		var/msg = ""
+		if(ticker.current_state == GAME_STATE_STARTUP)
+			msg = " (The server is still setting up, but the round will be \
+				started as soon as possible.)"
+		message_admins("<font color='blue'>\
+			[usr.key] has started the game.[msg]</font>")
```
Comment (XDTM) [inline code/modules/admin/admin.dm]: hide the line endings before they notice

**Comment by Firecage** [comment]
Ehhh, personally I would rather have this as a seperate verb for debugging use. I don't want to accidentally press it to early while on my local repo, nor would I want an admin to accidentally press it and fuck everything up.

File: code/modules/admin/admin.dm [hunk 2]
```diff
@@ -513,15 +513,17 @@ var/global/BSACooldown = 0
...
-	if(ticker.current_state == GAME_STATE_PREGAME)
-		ticker.can_fire = 1
-		ticker.timeLeft = 0
+	if(ticker.current_state == GAME_STATE_PREGAME || ticker.current_state == GAME_STATE_STARTUP)
+		ticker.start_immediately = TRUE
 		log_admin("[usr.key] has started the game.")
-		message_admins("<font color='blue'>[usr.key] has started the game.</font>")
+		var/msg = ""
+		if(ticker.current_state == GAME_STATE_STARTUP)
+			msg = " (The server is still setting up, but the round will be \
```
Comment (ChangelingRain) [inline code/modules/admin/admin.dm]: NO LINE BREAKS

Comment (RemieRichards) [inline code/modules/admin/admin.dm]: Coiax your defiance is cute, but cease.

Comment (coiax) [inline code/modules/admin/admin.dm]: Code where one line of code is split across mutiple lines **(except for strings and comments and in those cases existing longer lines must not be split up)**

Not an existing line.

**Comment by MrStonedOne** [comment]
Feature.

**Comment by Core0verload** [comment]
MSO stop your memes already.

**Comment by PKPenguin321** [comment]
For the purposes of a code freeze, this would probably fall under Feature

**Pull request merged**